### PR TITLE
adding first draft of intro blog

### PR DESCRIPTION
**Pull request by mikemunsell**

### CODE REVIEW

File: _posts/2016-06-24_mikemunsell_intro.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Introduction Post"
```
Comment (arokem) [inline _posts/2016-06-24_mikemunsell_intro.md]: Please use "Introducting DSSG Fellow Mike Munsell"

Comment (arokem) [inline _posts/2016-06-24_mikemunsell_intro.md]: And by "introducting", I mean "introducing"

File: _posts/2016-06-24_mikemunsell_intro.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+date:   2016-06-24
+author: Mike Munsell
+---
+
+###Background
```
Comment (arokem) [inline _posts/2016-06-24_mikemunsell_intro.md]: To be on the safe side, I would put a space between the hashes and the words in your sub-titles

File: _posts/2016-06-24_mikemunsell_intro.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+It is a very exciting time to be a researcher in my field, particularly given the unlocked potential of large datasets in advancing the science of econometric forecasting and policy decision-making.  DSSG will provide me with exposure to solving real-world problems with tools that are directly applicable to economic theory and computation.  I hope to take the skills that I learn this summer and apply it to my own research and collaborative work when I return to Boston in the fall.
+
+###What Do I Want to Learn?
+I'd like to brush up on my programming skills (particularly since I spent the last few months only studying theory to pass my field exams!), as well as expand my knowledge of machine learning techniques.  Additionally, I hope that my time spent in the DSSG program will help me develop research questions that have both societal and data-intensive implications.
+
+###What Have You Learned So Far
```
Comment (arokem) [inline _posts/2016-06-24_mikemunsell_intro.md]: To be consistent with your other sub-titles, please change this to the first person ("What have I learned so far?")

**Comment by arokem** [comment]
I had a few comments, and ping me when you want me to merge this (I'm being careful, because you labeled this as a "draft").

File: _posts/2016-06-24_mikemunsell_intro.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+It is a very exciting time to be a researcher in my field, particularly given the unlocked potential of large datasets in advancing the science of econometric forecasting and policy decision-making.  DSSG will provide me with exposure to solving real-world problems with tools that are directly applicable to economic theory and computation.  I hope to take the skills that I learn this summer and apply it to my own research and collaborative work when I return to Boston in the fall.
+
+### What Do I Want to Learn?
+I'd like to brush up on my programming skills (particularly since I spent the last few months only studying theory to pass my field exams!), as well as expand my knowledge of machine learning techniques.  Additionally, I hope that my time spent in the DSSG program will help me develop research questions that have both societal and data-intensive implications.
+
+### What Have I Learned So Far
```
Comment (arokem) [inline _posts/2016-06-24_mikemunsell_intro.md]: One more tiny nit to pick: I am not an English native speaker, but maybe this should be "What have I learned so far?" or "What I have learned so far"?

Comment (arokem) [inline _posts/2016-06-24_mikemunsell_intro.md]: That is: one is a question, and the other is a simple statement

**Comment by mikemunsell** [comment]
Great point!  I have changed it

**Pull request merged**